core/network/protocols/sup_protocol.cpp

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "network/protocols/sup_protocol.hpp"
+
+#include "network/common.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::network, SupProtocol::Error, e) {
```

Comment:
There errors seem to be duplicated across every protocol, would it make sense to extract them into a separate file, common for all protocols?

Reply:
Hm... It makes sence. Thanks.